content/cloud/cloud.md

```diff
@@ -0,0 +1,22 @@
+---
+Title: 'Cloud Computing'
+Description: 'Cloud Computing refers to an architecture where computing resources and hardware are maintained and managed remotely by an entity other than the user.'
+Codecademy Hub Page: 'https://www.codecademy.com/catalog/subject/cloud-computing'
+CatalogContent:
+  - 'learn/foundations-of-cloud-computing'
+  - 'paths/back-end-engineer-career-path'
+---
+
+**Cloud Computing** refers to an architecture where computing resources and hardware are maintained and managed remotely by an entity other than the user. This is becoming more popular with businesses, and has even become ubiquitous in personal computing. Whenever a person sets up a website on GoDaddy, plays _World of Warcraft_, or streams a movie on Netflix, they are using a cloud computing solution.
+
+## Advantages
+
+Cloud computing offers the following advantages to the user:
+
+- Scalability: Adding resources is quicker and easier in a cloud environment then in an on-prem data center which involves purchasing, installing and configuring new hardware.
+- Security: A cloud solution often provides more security resources than a business, especially a small business, can manage on their own.
+- Availability: Most cloud solutions have more robust and redundant architecture than a small to medium business can afford, leading to better uptime for users and clients.
+- Performance: A cloud solution will provide more powerful hardware than a business could afford in its own data center.
+- Cost: A cloud solution is typically more affordable than an on-prem solution offering the same capabilities.
+
+Generally, cloud computing removes the necessity of a business to maintain a dedicated data center, with all the hardware and personnel it requires, for a particular solution, allowing the business to allocate those resources to other tasks. Because of this, cloud solutions are increasing in popularity, especially with the increasing cost of keeping hardware and software up-to date.
```

Comment:
```suggestion
Generally, cloud computing removes the necessity of a business to maintain a dedicated data center, with all the hardware and personnel it requires, for a particular solution, allowing the business to allocate those resources to other tasks. Because of this, cloud solutions are increasing in popularity, especially with the increasing cost of keeping hardware and software up to date.
```